### DISCUSSION TITLE
How do you run TSLint so that it only outputs Errors?

### DISCUSSION THREAD

**Discussion by Splaktar** [nrwl/nx#4938] [General]
Ideally, I need something like `nx affected:lint --parallel --maxParallel=16 --quiet` to work, but it's still showing a lot of warnings. So many that CircleCI won't display the output and makes you download the file to see the errors.

For CI, I only want to show the errors. I saw a PR get merged that added `--quiet` support for ESLint, but does that support exist for TSlint (the [TSLint CLI](https://palantir.github.io/tslint/usage/cli/) has a `--quiet` option) too?

I saw on https://nx.dev/latest/angular/cli/lint that there is a `--silent` option, but that says that it `Hide output text.` which isn't what I want. I just want it to hide everything but errors. I did test it and verified that it actually hides errors in addition to warnings.

I didn't see anything related on https://nx.dev/latest/angular/cli/affected-lint.

This https://github.com/nrwl/nx/issues/3098 seems related and was implemented in PR https://github.com/nrwl/nx/pull/3108. https://nx.dev/latest/angular/linter/lint#quiet seems to indicate that this option exists somewhere, I'm just not sure how to get to it from the CLI.

**Comment by Splaktar** [comment]
Hrm, I guess it's probably because our lint targets are still configured with
```json
"builder": "@angular-devkit/build-angular:tslint",
```
I tried to see if I could use the `@nrwl/linter` builder here, but it looks like TSLint isn't supported at all now. It has `@nrwl/linter:eslint` and `@nrwl/linter:lint` (which points at `eslint` now too).

**Comment by juddgaddie** [comment]
related to #11729
